OOModel/src/declarations/Class.cpp

```diff
@@ -228,4 +239,22 @@ Class* Class::expressionToClass(Expression* expr)
 	return nullptr;
 }
 
+QSet<Class*> Class::directSubClasses()
+{
+	auto top = root();
+	//Find all the classes where this method is subclasses
+	QSet<Class*> result;
+	QList<Model::Node*> toCheck;
+	toCheck.append(top);
+	while (!toCheck.isEmpty())
```

Comment:
Class nodes could have inner classes. We could even have inner classes inside methods (not really a Java or C++ thing, but Envision "supports" that). I guess we'll leave it for now. At some point in the future we'll anyway need better reference handling, so there might be a reverse reference map somewhere.